pkg/workflow/common/cluster.go

```diff
@@ -17,3 +17,15 @@ func GetExecutionClusterClient(wfr *v1alpha1.WorkflowRun) kubernetes.Interface {
 
 	return store.GetClusterClient(cluster)
 }
+
+// IsMaster judge weather this wkf in master cluster or not
```

Comment:
`IsMaster judge whether this wfr will be executed in the master cluster or not` is better?